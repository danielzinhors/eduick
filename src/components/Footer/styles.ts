import styled, { css } from 'styled-components'
import media from 'styled-media-query'

export const Wrapper = styled.footer`
  ${({ theme }) => css`
    background: ${theme.colors.primary};
    height: 9.8rem;
    ${media.greaterThan('medium')`
      height: 10rem;
    `}
    ${media.greaterThan('huge')`
      height: 30rem;
    `}
  `}
`
