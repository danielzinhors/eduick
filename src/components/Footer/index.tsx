import * as S from './styles'

const Footer = () => <S.Wrapper aria-label="footer"></S.Wrapper>

export default Footer
