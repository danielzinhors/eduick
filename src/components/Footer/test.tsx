// import { screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/tests/helpers'

import Footer from '.'

describe('<Footer />', () => {
  it('should render 4 column topics', () => {
    const { container } = renderWithTheme(<Footer />)

    // expect(screen.getByText(/footer/i)).toHaveStyle({
    //   color: '#200E64'
    // })

    expect(container.firstChild).toMatchSnapshot()
  })
})
