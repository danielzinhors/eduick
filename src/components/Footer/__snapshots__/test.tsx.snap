// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 4 column topics 1`] = `
.c0 {
  background-color: #200E64;
}

<footer
  aria-label="footer"
  class="c0"
/>
`
